apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/PooledSession.cs

```diff
@@ -152,18 +166,13 @@ internal PooledSession WithTransaction(ByteString transactionId, ModeOneofCase t
         /// session. For instance, when retrying aborted commits it is better if the transaction work and commit
         /// are retried with the same session, because after each abort the sessions' lock priority increments.
         /// </remarks>
-        /// <returns>A new instance of <see cref="PooledSession"/>.</returns>
-        /// <exception cref="InvalidOperationException">If this <see cref="PooledSession.TransactionMode"/>
-        /// is <see cref="ModeOneofCase.None"/>.</exception>
-        public Task<PooledSession> WithFreshTransactionOrNewAsync(TransactionOptions transactionOptions, CancellationToken cancellationToken)
+        public Task<PooledSession> WithFreshTransactionOrNewAsync(CancellationToken cancellationToken)
```

Comment:
I'm really hoping this either goes away in a later commit, or makes more sense to me.